src/pe/header.rs

```diff
@@ -569,6 +569,29 @@ impl Header {
             optional_header,
         })
     }
+
+    pub fn parse_without_dos(bytes: &[u8]) -> error::Result<Self> {
```

Comment:
needs documentation if `pub`; i also want to understand what's different here and why this can't be the original function for parsing?

Reply:
@m4b Hello! This bug occurs when parsing file AmdCleanupUtility https://www.amd.com/en/resources/support-articles/faqs/GPU-601.html. Should I add `.exe` file to tests?

Reply:
> need to understand why you need the new pub function

Do you mean error on `DosHeader::parse(&bytes)?` should be handled inside original `Header::parse`?
And try to parse without DosHeader in case of Error?

Reply:
What I'm asking is why is a new function being added, when does the user call it, and why can't we handle this transparently inside of the regular `parse` function? Ideally I'd prefer not to add another `pub` function if possible; maybe it is not possible, I don't know

Reply:
we can handle this transparently inside of the regular `parse` function, but
1. users may expect that `parse` returns error when DOS stub is missing
2. there are existing tests that check invalid stub will not parse
3. when it is known that we have not DOS stub in buffer, it makes sense to use `parse_without_dos`

Reply:
I just added the documentation, is it ok now?

Reply:
ok thank you for clarifying when this happens, and why to use; i've suggested grammar for the documentation of `parse_without_dos`